app/pages/statistics.tsx

```diff
@@ -0,0 +1,304 @@
+/* eslint-disable visualize-admin/no-large-sx */
+import { Box, Card, Tooltip, Typography } from "@mui/material";
+import { max, rollups, sum } from "d3-array";
+import { timeFormat } from "d3-time-format";
+import uniq from "lodash/uniq";
+import { GetServerSideProps } from "next";
+import { ComponentProps, useMemo } from "react";
+
+import { AppLayout } from "@/components/layout";
+import { BANNER_MARGIN_TOP } from "@/components/presence";
+import { getAllConfigsMetadata } from "@/db/config";
+import { Serialized, deserializeProps, serializeProps } from "@/db/serialize";
+import { flag } from "@/flags";
+
+type PageProps = {
+  configsMetadata: Awaited<ReturnType<typeof getAllConfigsMetadata>>;
+};
+
+export const getServerSideProps: GetServerSideProps<PageProps> = async () => {
+  return {
+    props: serializeProps({
+      configsMetadata: (await getAllConfigsMetadata()).sort((a, b) => {
+        return b.created_at.getTime() - a.created_at.getTime();
+      }),
+    }),
+  };
+};
+
+const Statistics = (props: Serialized<PageProps>) => {
+  const { configsMetadata } = deserializeProps(props);
+  const countByDate = useMemo(
+    () => getCountByDate(configsMetadata),
+    [configsMetadata]
+  );
+  const total = sum(countByDate, (d) => d[1].count) ?? 0;
+  const averageChartCountPerMonth = Math.round(total / countByDate.length);
+  const now = Date.now();
+  const nowDate = new Date(now);
+  const lastMonthDailyAverage =
+    configsMetadata.filter(({ created_at: date }) => {
+      return date > new Date(now - 30 * DAYS_IN_MS) && date <= nowDate;
+    }).length / 30;
+  const previousThreeMonthsDailyAverage =
+    configsMetadata.filter(({ created_at: date }) => {
+      return date > new Date(now - 90 * DAYS_IN_MS) && date <= nowDate;
+    }).length / 90;
```

Comment:
Nit: The computation of stats could be behind a big memo, and applied directly database side ?

Helped from ChatGPT gave me :

```
SELECT
    DATE_TRUNC('month', created_at) AS month,
    COUNT(*) AS monthly_count
FROM
    config
WHERE
    created_at >= DATE_TRUNC('month', CURRENT_DATE) - INTERVAL '1 year'
GROUP BY
    DATE_TRUNC('month', created_at)
ORDER BY
    month;
```

```
        month        | monthly_count 
---------------------+---------------
 2023-06-01 00:00:00 |             1
 2023-07-01 00:00:00 |             5
 2023-08-01 00:00:00 |             3
 2023-09-01 00:00:00 |             3
 2023-10-01 00:00:00 |             8
 2023-11-01 00:00:00 |             1
 2023-12-01 00:00:00 |             5
 2024-01-01 00:00:00 |             4
 2024-02-01 00:00:00 |            17
```

Reply:
Optimized a bit in https://github.com/visualize-admin/visualization-tool/pull/1610/commits/5d48237f4f2d4998e5ab28888004ab194adfa12c 👍